src/Directory.Build.targets

```diff
@@ -7,6 +7,7 @@
       https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json;
       https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json;
       https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json;
+      $(RestoreSources)
```

Comment:
Is this intentional?

Reply:
yes, opportunistic to allow adding on for other projects (Angela discovered this and reported it as a bug to DevDiv)